scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/models/BundleConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.typesafe.conductr.clientlib.scala.models
+
+import java.net.URI
+
+// FIXME: Instead of using the name 'BundleConfigEndpoint' we should create the case class 'Endpoint'
+//        inside the companion object 'BundleConfig' to access it with 'BundleConfig.Endpoint'.
+//        However, when using 'BundleConfig.Endpoint' the Json.format in 'JsonMarshalling' doesn't work anymore!
+case class BundleConfigEndpoint(bindProtocol: String, services: Set[URI])
```

Comment:
This probably shoulnd't be a FIXME then... i.e. it is good to have the comment, but it isn't something we're probably going to fix.